cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -989,15 +957,13 @@ void ThinClientPoolDM::AddPdxType(std::shared_ptr<Serializable> pdxType,
 std::shared_ptr<Serializable> ThinClientPoolDM::GetPDXTypeById(int32_t typeId) {
   LOGDEBUG("ThinClientPoolDM::GetPDXTypeById:");
 
-  GfErrType err = GF_NOERR;
-
   TcrMessageGetPdxTypeById request(
       new DataOutput(m_connManager.getCacheImpl()->createDataOutput()), typeId,
       this);
 
   TcrMessageReply reply(true, this);
 
-  err = sendSyncRequest(request, reply);
+  auto err = sendSyncRequest(request, reply);
```

Comment:
How about we get rid of `err` here altogether?  Checking it against GF_NOERR then doing `throwExceptionIfError` when we already know it's an error seems redundant...

```
throwExceptionIfError("Operation Failed", sendSyncRequest(request, reply));
```